backend/src/components/user.js

```diff
@@ -9,11 +9,24 @@ async function getUserInfo(req, res) {
       message: 'No session data'
     });
   }
+
+  /* 
+  data used for dev build of LandingPage.vue 
+  will be replaced with API data at a later time
+      isDraft: false,
+      applicationSubmitted: true,
+      applicationApproved:
+  */
+
   let resData = {
     displayName: `${req.session.passport.user.displayName}`,
     organizationList: [{
       organizationName: 'ABC organization',
       organizationId: 'org123',
+      isDraft: false,
```

Comment:
can we consolidate isDraft, applicationSubmitted, applicatinApproved to be one status code with DRAFT, SUBMITTED, APPROVED

Reply:
done!